.gitignore

```diff
@@ -16,6 +16,7 @@ web/static/js/libs*.js
 
 config/active.dat
 config/config.json
+/plugins
```

Comment:
This is where the installed plugins will live. They won't be committed to the repo at this location.